cluster/src/main/java/com/hartwig/pipeline/calling/command/BcfToolsCommandBuilder.java

```diff
@@ -0,0 +1,81 @@
+package com.hartwig.pipeline.calling.command;
+
+import java.util.List;
+
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.Lists;
+import com.hartwig.pipeline.execution.vm.BashCommand;
+import com.hartwig.pipeline.execution.vm.unix.PipeCommands;
+
+public class BcfToolsCommandBuilder  {
```

Comment:
could use Immutables to reduce the builder boilerplate

Reply:
Can I have a bit more detail please. I can't quite figure what you mean.

Reply:
You could make a BclToolsBuilder interface, with an @Immutable annotation, and all the construction methods as interface methods. Then let Immutables generate all the internal state of the builder, etc. Only awkward bit is you'd probable need to create a special build method or something to return an actual command... not that fussed about it, just if you think its a good idea.

Reply:
Gotcha. Think i will leave it for this one.